packages/server/tests/MpeSearchEngine.test.ts

```diff
@@ -281,6 +285,298 @@ test.group('My MPE Rooms Search', (group) => {
     });
 });
 
+test.group("Other user's MPE Rooms Search", (group) => {
+    const {
+        createUserAndGetSocket,
+        disconnectEveryRemainingSocketConnection,
+        initSocketConnection,
+    } = initTestUtils();
+
+    group.beforeEach(async () => {
+        initSocketConnection();
+        await Database.beginGlobalTransaction();
+    });
+
+    group.afterEach(async () => {
+        await disconnectEveryRemainingSocketConnection();
+        sinon.restore();
+        await Database.rollbackGlobalTransaction();
+    });
+
+    test("It should send back all other user's public MPE rooms", async (assert) => {
```

Comment:
**Suggestion:** Unless I'm mistaken, the related user playlistVisibility is not covered here ?
In other words the method should be tested here `getIfUserCanQueryOtherUserMpeRooms` ?

Reply:
You're right, I'll add a test to ensure an error is thrown when querying the rooms of a user that disallows it (either by `PRIVATE` OR `FOLLOWERS_ONLY` prop).